Trunk.toml

```diff
@@ -0,0 +1,21 @@
+[extension]
+name = "pgmq"
+repository = "https://github.com/tembo-io/pgmq"
+license = "PostgreSQL"
+description = "A lightweight message queue for PostgreSQL, like AWS SQS or RSMQ, but entirely in the database."
+homepage = "https://github.com/tembo-io/pgmq"
+documentation = "https://github.com/tembo-io/pgmq"
+categories = [
+      "mq",
+      "message queue",
+      "messaging",
+      "queues",
+      "fifo",
+      "sqs",
+      "rsmq"
+]
```

Comment:
@vrmiguel what are the possible enums for these categories? can we put any string that we want?

Reply:
Copied those from META.json

Reply:
@ChuckHend the possible values are

```
            name             |           slug            
-----------------------------+---------------------------
 Tooling / Admin             | tooling_admin
 Metrics                     | metrics
 Change Data Capture         | change_data_capture
 Orchestration               | orchestration
 Procedural Languages        | procedural_languages
 Machine Learning            | machine_learning
 Analytics                   | analytics
 Connectors                  | connectors
 Data / Transformations      | data_transformations
 Index / Table Optimizations | index_table_optimizations
 Query Optimizations         | query_optimizations
 Auditing / Logging          | auditing_logging
 Debugging                   | debugging
 Featured                    | featured
 Search                      | search
 Security                    | security
```

Also available in https://registry.pgtrunk.io/categories/all

Reply:
Maybe "connectors"? But this seems to apply more to fdws... I don't think there's a single one that fully matches pgmq.

Reply:
It looks like we are currently using `["featured", "orchestration"]`.  https://pgt.dev/extensions/pgmq

Reply:
I gave wrong link. Here is where we are setting those categories
https://github.com/tembo-io/pgmq/blob/ced449fc6ac1bdf86d82c057653990de99fcd76b/.github/workflows/extension_ci.yml#L151

Reply:
Does --category override `Trunk.toml`? Should I maybe just remove the categories key from `Trunk.toml`?